### ISSUE TITLE
avifenc: Add an option to ignore the alpha channel

### ISSUE THREAD

**Issue by vigneshvg** [AOMediaCodec/libavif#1665]
Add an option to avifenc that will make it ignore the alpha channel even if it is found in the input. The option can be named `--ignore-alpha` or `--no-alpha`.

Some tools add alpha channels to PNG files inadvertently. So it would help `avifenc` encode such images without the alpha channel (see https://github.com/AOMediaCodec/libavif/issues/1658#issuecomment-1751388161 for an example).

**Comment by vrabaud** [comment]
We could also have an option to pre-multiply RGB with A.
FYI, WebP also has an option to blend the alpha against a given color.
```
  -blend_alpha <hex> ..... blend colors against background color
                           expressed as RGB values written in
                           hexadecimal, e.g. 0xc0e0d0 for red=0xc0
                           green=0xe0 and blue=0xd0
  -noalpha ............... discard any transparency information
```

**Comment by vigneshvg** [comment]
> We could also have an option to pre-multiply RGB with A. FYI, WebP also has an option to blend the alpha against a given color.
> 
> ```
>   -blend_alpha <hex> ..... blend colors against background color
>                            expressed as RGB values written in
>                            hexadecimal, e.g. 0xc0e0d0 for red=0xc0
>                            green=0xe0 and blue=0xd0
>   -noalpha ............... discard any transparency information
> ```

Yes, we could certainly add that option as well. This issue was just to ignore the alpha channel (similar to `-noalpha`) in webp to deal with files which inadvertently have an alpha channel in the input and have no use for them.

**Comment by wantehchang** [comment]
Fixed by PR https://github.com/AOMediaCodec/libavif/pull/3216. Added the --ignore-alpha option to avifenc.
